components/Tabs2/tests/snapshot/MDCTabBarViewItemViewSnapshotTests.m

```diff
@@ -54,4 +51,17 @@ - (void)testItemViewNoTitleNoImageIntrinsicContentSize {
   [self generateSnapshotAndVerifyForView:itemView];
 }
 
+- (void)testItemViewLongTitleRegularImageIntrinsicContentSize {
+  // Given
+  MDCTabBarViewItemView *itemView = [[MDCTabBarViewItemView alloc] init];
+  itemView.image = [UIImage mdc_testImageOfSize:CGSizeMake(24, 24)];
+  itemView.title = @"123456789012345678901234567890";
```

Comment:
From the test name, it looks like portions of the "Given" section should actually be in the "When" (test conditions).  The `generateSnapshot...` is usually the "Then" clause.

Reply:
Done.